### ISSUE TITLE
Heading Levels skipped

### ISSUE THREAD

**Issue by ainemckay** [MHRA/products#923]
## 1.3.1 Info and relationships (A)
Heading levels were skipped and /or a main heading (h1) was missing from the page.

## Recommendation
Review which text should be headings, and in general ensure that headings descend in order, creating sections and sub-sections.

For headings that are not visually represented, it is possible to create hidden headings which allows users of screen readers to understand the content on the page. 

## Acceptance criteria

- [x] Headers should follow semantic guidelines - one `<h1>` and then `<h2>`s and `<h3>`s are properly layered

## Testing instructions
- Go to non-prod site
- Click on index letter (e.g. "A") and inspect DOM to ensure it's a `<h2>`
- Search for a drug in the search box (e.g. "ibuprofen") and accept the disclaimer
- Check the "Showing results for..." is a `<h2>`
- Check the hidden "Search results" `<h3>` above the search results
- Do a ctrl+f and search for "h1", "h2" and "h3" to ensure they follow the proper order on the page
- View on production site to see old, non-compliant structure
